.github/copilot-instructions.md

```
@@ -89,7 +98,18 @@ applyTo: "**"
 - **Internationalization**: Prefer `react-i18next`. All production UI must use `useTranslation()` keys; validate keys with the translation script.
 
 ## Monitoring and analytics
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)